src/main/java/org/jabref/logic/integrity/PersonNamesChecker.java

```diff
@@ -33,13 +33,16 @@ public PersonNamesChecker(BibDatabaseContext databaseContext) {
 
         // Remove all brackets to handle corporate names correctly, e.g., {JabRef}
         value = new RemoveBrackets().format(value);
+
+        // put ' ' after every name, e.g.,L.C. Brand -> L. C. Brand
+        String checkFormat = value.replaceAll("\\.", ". ").replaceAll("\\  ", " ");
```

Comment:
Why do we need `replaceAll("\\ ", " ")`

Reply:
If we don't need `replaceAll("\\ ", " ")` , for example, `L.C. Brand` ,`checkFormat` will be   `L. C. Briand` ( there are two spaces between `L.` and `C.` , `C.` and `Brand`, not one space). So there's also a warning message. We use `checkFormat` to help check the name format.